behaviors/README.md

```diff
@@ -104,6 +104,21 @@ sideBarArea:
 
 Component lists don't use `_label` or `_display` (and will ignore them if you specify them), but they allow `_placeholder`. It's recommended to add placeholders to component lists, which will display when that list is empty.
 
+##### Site-specific Components
+
+You may optionally specify that components in a list should be included or excluded on a specific site. To do so, add a comma-separated list of sites in parenthesis after the name of the component in the `include` list:
+
+```yaml
+include:
+  - article (site1, site2) # allow on site1 and site2 ONLY
+  - paragraph # allow on all sites
+  - image (not:site1) # allow on all sites EXCEPT site1
+  - link (site1, site2, site3, not:site2, not:site3) # only allow on site1
```

Comment:
as a less-used +1, I think this is fine, especially since it's only for implementation-specific components (adding specific sites to a list in a component on npm would be dumb)